command.go

```diff
@@ -5114,6 +5114,7 @@ type ClientInfo struct {
 	OutputListLength   int           // oll, output list length (replies are queued in this list when the buffer is full)
 	OutputMemory       int           // omem, output buffer memory usage
 	TotalMemory        int           // tot-mem, total memory consumed by this client in its various buffers
+	IoThread           int           // io-thread id
```

Comment:
Could you please add a test case to verify a new field in response schema?

Reply:
The integration test in https://github.com/redis/go-redis/blob/9f1522a91e666ee40f807945e49396b09e4f87c2/example_test.go#L178 will actually test client info parser. Exactly having that test failing pointed me to the missing key.

Reply:
Ah, okay! Counts as testing as well